includes/admin.form.inc

```diff
@@ -31,6 +31,12 @@ function islandora_large_image_admin(array $form, array &$form_state) {
       '#default_value' => $get_default_value('islandora_lossless', FALSE),
       '#description' => t('Lossless derivatives are of higher quality but adversely affect browser performance.'),
     ),
+    'islandora_convert_rgb8' => array(
```

Comment:
Would be better to have variables prefixed with Module name. Not set in stone but good practice. Also needs to be dealt with on the .install file (remove on uninstall)